apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -477,6 +477,110 @@ const AutoSize = () => {
   );
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28649&commitId=c8df88b5197eedf6642d5acc64752109fc4fa95b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Positioning</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size with async content reset styles on updatePosition.chromium.png|0| Added|
|Positioning.auto size with async content reset styles on updatePosition.floating element is fully visible.chromium.png|0| Added|
|Positioning.auto size with async content.chromium.png|0| Added|
|Positioning.auto size with async content.floating element is within the boundary.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>